--- pungi-2.0.22/src/bin/pungi.py.isoparam
+++ pungi-2.0.22/src/bin/pungi.py
@@ -149,7 +149,7 @@ if __name__ == '__main__':
             config.set('pungi', option.dest, value)
             # When setting name, also set the iso_basename.
             if option.dest == 'name':
-                config.set('pungi', 'iso_basename', value)
+                config.set('pungi', 'iso_basename', value.replace(' ', '_'))
 
         # Pulled in from config file to be cli options as part of pykickstart conversion
         parser.add_option("--name", dest="name", type="string",
--- pungi-2.0.22/src/pypungi/__init__.py.isoparam
+++ pungi-2.0.22/src/pypungi/__init__.py
@@ -1121,12 +1121,8 @@ cost=500
             extraargs.extend(sparcbootargs)
 
         extraargs.append('-V')
-        if treesize > 700:
-            extraargs.append('%s %s %s DVD' % (self.config.get('pungi', 'name'),
-                self.config.get('pungi', 'version'), self.config.get('pungi', 'arch')))
-        else:
-            extraargs.append('%s %s %s' % (self.config.get('pungi', 'name'),
-                self.config.get('pungi', 'version'), self.config.get('pungi', 'arch')))
+        extraargs.append('%s %s %s' % (self.config.get('pungi', 'name'),
+            self.config.get('pungi', 'version'), self.config.get('pungi', 'arch')))
 
         extraargs.extend(['-o', isofile])
         
